### PR TITLE
FF108 Relnote: CSP directive [script|style]-src-[elem|attr]

### PR DESCRIPTION
**Pull request by hamishwillee**
FF108 ships [style-src-elem](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src-elem), [style-src-attr](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src-attr), [script-src-elem](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src-elem), [script-src-attr](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src-attr) in [bugzilla1782513](https://bugzilla.mozilla.org/show_bug.cgi?id=1782513).

This adds the release note. Other docs work can be tracked in #22113

### CODE REVIEW

**Comment by bsmth** [comment]
Thanks Hamish, taking a look 👀

File: files/en-us/mozilla/firefox/releases/108/index.md [hunk 1]
```diff
@@ -34,6 +34,11 @@ This article provides information about the changes in Firefox 108 that will aff
 
 ### HTTP
 
+- [`Content-Security-Policy`](/en-US/docs/Web/HTTP/Headers/Content-Security-Policy) HTTP header directives [`style-src-elem`](/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src-elem) and [`style-src-attr`](/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src-attr) are now supported.
+  A server can use these specify valid sources for stylesheet `<style>` elements and `<link>` elements with `rel="stylesheet"`, and for styles applied to individual elements, respectively ({{bug(1529338)}}).
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/108/index.md:38]: Minor typo fix:

```suggestion
  A server can use these to specify valid sources for stylesheet `<style>` elements and `<link>` elements with `rel="stylesheet"`, and for styles applied to individual elements, respectively ({{bug(1529338)}}).
```

**Review by bsmth — APPROVED** [review]
Thanks Hamish, merging shortly 👍🏻

**Comment by hamishwillee** [comment]
Thanks for the review!

**Pull request merged**